lib/charms/mongodb_libs/v0/mongodb.py

```diff
@@ -52,6 +52,8 @@ class MongoDBConfiguration:
     password: str
     hosts: Set[str]
     roles: Set[str]
+    tls_external: bool
```

Comment:
update docstring (L42-L48) to match all fields

Reply:
good eye!